internal/pkg/views/statusbar.go

```diff
@@ -124,11 +124,11 @@ func (w *StatusbarWidget) Layout(g *gocui.Gui) error {
 	}
 
 	if w.currentMessage.InProgress {
-		fmt.Fprint(v, style.Loading("⏳  "+w.currentMessage.Message))
+		fmt.Fprint(v, style.Loading(w.currentMessage.Icon()+"  "+w.currentMessage.Message))
```

Comment:
This condition has been moved into the `Icon()` method so arguably isn't required (atleast for the `InProgress` and `Failure`, might still want then to apply a different style to `Completed`)

Reply:
Good spot - will fix up. Thanks!

Reply:
Done - thanks!